service/src/main/java/greencity/service/ubs/ValuesForUserTableServiceImpl.java

```diff
@@ -76,7 +76,7 @@ private UserWithSomeOrderDetailDto mapToDto(User u) {
         if (optional.isPresent()) {
             allFieldsFromTableDto
                 .setLastOrderDate(optional
-                    .get().getOrderDate().toLocalDate().format(DateTimeFormatter.ofPattern("dd/MM/yyyy")));
+                    .get().getOrderDate().toLocalDate().format(DateTimeFormatter.ofPattern("yyyy-MM-dd")));
```

Comment:
extract "yyyy-MM-dd" into constant and reuse

Reply:
fixed